packages/react-native-cli/src/commands/InstallCommand.ts

```diff
@@ -44,11 +44,14 @@ async function installJavaScriptPackage (projectRoot: string): Promise<void> {
 async function addGradlePluginDependency (projectRoot: string): Promise<void> {
   logger.info('Adding the Bugsnag Android Gradle Plugin')
 
+  const androidPluginVersion = await findAndroidPluginVersion(projectRoot)
+  const suggestedPluginVersion = androidPluginVersion?.startsWith('7.') ? '7+' : '5+'
```

Comment:
what happens when AGP gets to 8? I wonder if we can future proof this a little?

Reply:
We could emit a suggestion for `8+` in that case, but then we could (in theory) also start suggesting versions before they are released.

Reply:
How about going the other way and only suggesting a version if we positively match it? i.e. `5+` if the version is <7, `7+` if the version =7, and nothing otherwise. We could print a suitable warning to say that we couldn't find an AGP version that we support and link to the docs?

(The message below needs updating either way.)